src/esp/nav/PathFinder.cpp

```diff
@@ -134,6 +145,7 @@ class IslandSystem {
         if (navMesh->isValidPolyRef(startRef) &&
             (polyToIsland_.find(startRef) == polyToIsland_.end())) {
           uint32_t newIslandId = islandRadius_.size();
+          islandsToPolys_[newIslandId] = std::vector<dtPolyRef>();
```

Comment:
This is inefficient. Since we don't support C++17, unfortunately, the proper way to do this is a bit ugly:
```suggestion
          islandsToPolys.emplace(std::piecewise_construct,
                                                 std::forward_as_tuple(newIslandId),
                                                 std::tuple<>());
```